### ISSUE TITLE
Better default frequency vector choice for unevenly-spaced records 

### ISSUE THREAD

**Issue by CommonClimate** [LinkedEarth/Pyleoclim_util#536]
Consistently, my experience has been that the spectral analysis of unevenly-spaced records results in spectra that are heavily distorted,  regardless of the spectral method (LS or WWZ), and  with all  currently existing `freq_method` options. 

Here is a minimal working example:
```
import pyleoclim as pyleo
LR04 = pyleo.utils.load_dataset('LR04')
psd_ls = LR04.standardize().spectral()
psd_sig = psd_ls.signif_test(number=100)
fig, ax = psd_sig.plot()
```
 The result shows a singularity near the Nyquist frequency:
![freaqy_spectrum](https://github.com/LinkedEarth/Pyleoclim_util/assets/13760223/eb1f977a-6beb-4c0a-8b8e-409ef3140bf0)

Passing a custom vector can help, but it requires knowing a fair bit about the jargon (OFAC, HIFAC, etc) AND the AR(1) spectra still don't look like what they are supposed to (i.e. tapering off at low frequencies).  I suspect that enough tinkering can solve the problem, but that is not fair to expect of our users. There has to be a better way. 

Proposal: either better defaults for the choice of `freq_method`, or a tutorial that explicitly addresses this.

**Comment by CommonClimate** [comment]
Telecon 04/12 decisions:
- [ ] fix utils.spectral.lomb-scargle() to avoid returning nonsensical low freq
- [ ] add to the spectral PyleoTutorial a section on the importance of the frequency vector for unevenly spaced series
- [ ] generate freq vector from MTM on interpolated series

**Comment by CommonClimate** [comment]
addressed by #540
